ahoopes16/tidkinbrae
# Replace Picture with Real Photo of Me

### Summary
This PR is to replace the fake blobfish picture with a real picture of yours truly. This PR closes #23.

If accepted, this PR will:
- Add a new version of my photo
- Replace the photo of the blobfish from the internet with said photo

### To Test
1. `yarn && yarn start`
1. Go to http://localhost:8000/
1. You should see a picture of me and my fiancee instead of a picture of a blobfish
